imagekitio/url.py

```diff
@@ -211,3 +212,20 @@ def transformation_to_str(transformation):
             )
 
         return Default.CHAIN_TRANSFORM_DELIMITER.value.join(parsed_transforms)
+
+    @staticmethod
+    def custom_encodeURIComponent(url_str):
```

Comment:
Please rename it to "encodeURI" because the functionality is similar to "encodeURI" in javascript and not "encodeURIComponent".

Reply:
done